.github/workflows/ci.yml

```diff
@@ -96,7 +96,7 @@ jobs:
           max_attempts: 5
           retry_wait_seconds: 30
           command: |
-            composer update --optimize-autoloader --no-interaction --no-progress ${{ matrix.composer-flags }}
+            composer update --no-interaction --no-progress --optimize-autoloader ${{ matrix.composer-flags }}
```

Comment:
what's the difference?

Reply:
None, only sorted alphabetically (as I was trying different things before squashing).

Reply:
i don't see value of changing the order, and nobody will manually maintain the alphabetical order of cli params.